pallets/phala/src/stakepool.rs

```diff
@@ -598,9 +603,12 @@ pub mod pallet {
 				Error::<T>::WorkerDoesNotExist
 			);
 			let miner: T::AccountId = pool_sub_account(pid, &worker);
-			mining::pallet::Pallet::<T>::start_mining(miner, stake)?;
+			mining::pallet::Pallet::<T>::start_mining(miner.clone(), stake)?;
 			pool_info.free_stake -= stake;
 			StakePools::<T>::insert(&pid, &pool_info);
+			// Save the preimage of the sub-account when start mining. We remove the storage item
+			// after reclaiming. Only start_mining is paired with reclaim. So we insert it here.
+			SubAccountPreimages::<T>::insert(miner, (pid, worker));
```

Comment:
Do you think we should have a event here? so indexer easy to know that the preimage created